src/agent/src/rpc.rs

```diff
@@ -201,7 +212,9 @@ impl AgentService {
         let p = if oci.process.is_some() {
             Process::new(
                 &sl!(),
-                oci.process.as_ref().unwrap(),
+                oci.process
+                    .as_ref()
+                    .ok_or_else(|| anyhow!("cannot get OCI process"))?,
```

Comment:
We've checked just above that `oci.process.is_some()`.  Instead we should use an `if let` to avoid the `unwrap()`.